src/components/FlyoutMenu/_flyout-menu.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../globals/vars';
+@import '~carbon-components/scss/globals/scss/layout';
+
+.#{$iot-prefix}-flyout-menu {
+  color: initial;
+  background: $ui-background;
+  min-width: 300px;
```

Comment:
I believe there are some use cases floating around from the design team that call for a flyout smaller than 300px width.

```suggestion
```